assets/js/components/optin.js

```diff
@@ -52,9 +53,25 @@ class Optin extends Component {
 			method: 'POST',
 		} )
 			.then( () => {
+				if ( !! checked && ! this.state.scriptOnPage ) {
+					const { jQuery: $ } = window;
```

Comment:
It's a bit more effort to handle without jQuery, though admittedly not much else in the plugin uses it. I did because it was there: but totally fair, I'll remove it.

Reply:
Turns out it wasn't even much more code 👍